src/v/cluster/tm_stm.h

```diff
@@ -65,6 +65,11 @@ struct tm_transaction {
         model::term_id etag;
     };
 
+    struct tx_group {
+        kafka::group_id group_id;
+        model::term_id etag;
```

Comment:
what does `e` stand for in `etag` maybe a comment above

Reply:
I think it stands for entity tag but frankly I didn't give it a thought, just used [microsoft jargon](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.resource.etag?view=azure-dotnet):

> ETags are used for concurrency checking when updating resources.